src/Symfony/Component/Intl/Regions.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Intl;
+
+use Symfony\Component\Intl\Exception\MissingResourceException;
+
+/**
+ * Gives access to region-related ICU data.
+ *
+ * @author Bernhard Schussek <bschussek@gmail.com>
+ * @author Roland Franssen <franssen.roland@gmail.com>
+ */
+final class Regions extends AbstractStaticBundle
+{
+    /**
+     * @return string[]
+     */
+    public static function getCodes(): array
```

Comment:
```suggestion
    public static function getRegionCodes(): array
```

for reasons listed above